srcpkgs/budgie-desktop-view/template

```diff
@@ -1,8 +1,9 @@
 # Template file for 'budgie-desktop-view'
 pkgname=budgie-desktop-view
 version=1.3
-revision=1
+revision=2
 build_style=meson
+configure_args="-D werror=false" #Dirty fix to budgie-desktop-view not compiling on meson 1.4.0 due to upstream errors
```

Comment:
This doesn't warrant a comment, just specify the argument.

Reply:
Alrighty, removed it.